.github/workflows/release_draft.yml

```diff
@@ -43,8 +43,6 @@ jobs:
         with:
           github-repository: ${{ github.repository }}
           version-tag: ${{ github.event.inputs.tag-name }}
-          branch: 'master'
```

Comment:
Why removed?

Reply:
The decision was made that the action would take care only of the validity of tags received from the GH API. There is no easy way to get a list of tags for a branch, so the branch is no longer part of this action.

This has been done in previous Release. Here I am only cleaning.